packages/blockchain-wallet-v4-frontend/src/data/components/brokerage/sagas.ts

```diff
@@ -325,7 +328,17 @@ export default ({
               dwStep: BankDWStepType.DEPOSIT_CONNECT
             })
           )
-          yield retry(100, 10000, ClearedStatusCheck, data.paymentId)
+          try {
+            const updatedOrder: SBTransactionType = yield retry(
+              30,
+              10000,
```

Comment:
(minor) I would move all of those magic numbers in self explained constants we can reuse them on several places inside this file and maybe around the app :)